public/Home.html

```diff
@@ -0,0 +1,44 @@
+<head>
+<title>marleigh - Home</title>
+</head>
+
+<body>
+<h1>HOME</h1>
+<br>
+<br>
+<br>
+<nav>
+  <nav>
```

Comment:
You've got an extra `<nav>` wrapper here. Wrapping these links in `<nav>` is great; gives them semantic relevance and leaves you a hook for styling later (today?).